docker/devel:openQA:ci/base/Dockerfile

```diff
@@ -1,5 +1,8 @@
 #!BuildTag: base
 #!UseOBSRepositories
+# preserve layers to prevent circleCI failing to download single big blobs,
```

Comment:
```suggestion
# preserve layers to avoid failures in CircleCI while it downloads a big blob in one attempt
```

We are *avoiding* a problem. Also I'd try and convey that it only tries once.

Reply:
done